### PR TITLE
add to path instructions

### PR DESCRIPTION
**Pull request by cboydstun**
tried to make things friendlier to newcomers =)

### CODE REVIEW

File: main/getting-started/before-using-agoric.md [hunk 1]
```diff
@@ -15,42 +16,53 @@ git clone https://github.com/Agoric/agoric-sdk
...
 
-Then proceed to [starting a project](/getting-started/start-a-project.md).
+### Add the `agoric` command to your PATH
+
+The `agoric` command is a shell script that uses `yarn` to run the Agoric SDK from the `agoric-sdk` directory. To use it from any directory on MacOS, add it to your path:
+
+```shell
+yarn link-cli ~/bin/agoric # creates an Agoric directory to reference
+echo $PATH # show current PATH
+export PATH=$PATH:/Users/*USERNAME*/bin # adds the parent directory to your PATH
```
Comment (dckc) [inline main/getting-started/before-using-agoric.md:28]:
```suggestion
export PATH=$PATH:$HOME/bin # adds the parent directory to your PATH
```

File: main/getting-started/before-using-agoric.md [hunk 2]
```diff
@@ -15,42 +16,53 @@ git clone https://github.com/Agoric/agoric-sdk
...
-agoric --version
 ```
 
-Then proceed to [starting a project](/getting-started/start-a-project.md).
+### Add the `agoric` command to your PATH
+
+The `agoric` command is a shell script that uses `yarn` to run the Agoric SDK from the `agoric-sdk` directory. To use it from any directory on MacOS, add it to your path:
+
+```shell
+yarn link-cli ~/bin/agoric # creates an Agoric directory to reference
```
Comment (dckc) [inline main/getting-started/before-using-agoric.md:26]: That assumes `~/bin` exists. And this step doesn't create any directories. How about:

```suggestion
mkdir -p ~/bin
yarn link-cli ~/bin/agoric # installs a small "agoric" script
```

File: main/getting-started/before-using-agoric.md [hunk 3]
```diff
@@ -15,42 +16,53 @@ git clone https://github.com/Agoric/agoric-sdk
 cd agoric-sdk
 yarn install
 yarn build
-yarn link-cli ~/bin/agoric
-agoric --version
 ```
 
-Then proceed to [starting a project](/getting-started/start-a-project.md).
+### Add the `agoric` command to your PATH
```
Comment (dckc) [inline main/getting-started/before-using-agoric.md:21]: Adding a whole section here is somewhat awkward.

How about we just add:

```
export PATH=~/bin:$PATH
```

before `agoric --version` in this section, and then move the whole dedicated section on PATH to the expanded directions below?

File: main/getting-started/before-using-agoric.md [hunk 4]
```diff
@@ -15,42 +16,53 @@ git clone https://github.com/Agoric/agoric-sdk
...
+
+The `agoric` command is a shell script that uses `yarn` to run the Agoric SDK from the `agoric-sdk` directory. To use it from any directory on MacOS, add it to your path:
+
+```shell
+yarn link-cli ~/bin/agoric # creates an Agoric directory to reference
+echo $PATH # show current PATH
+export PATH=$PATH:/Users/*USERNAME*/bin # adds the parent directory to your PATH
+echo $PATH # show updated PATH to include the parent directory of Agoric folder
+# RESTART YOUR SHELL FOR CHANGES TO TAKE EFFECT #
```
Comment (dckc) [inline main/getting-started/before-using-agoric.md:30]: Restarting your shell will make these changes disappear, I think.

File: main/getting-started/before-using-agoric.md [hunk 5]
```diff
@@ -15,42 +16,53 @@ git clone https://github.com/Agoric/agoric-sdk
...
+agoric --version # should print the version number of the SDK
+```
+
+Now you are ready proceed to [starting a project](/getting-started/start-a-project.md).
 
-A more detailed explanation follows.
+If `agoric` is not found, then you may need to add the parent directory of where your `yarn link-cli` command created a folder. For example, if you created a folder called `Agoric` in your `/Users/bin` directory, you may need to add `/Users/bin` to your PATH.
```
Comment (dckc) [inline main/getting-started/before-using-agoric.md:36]:
```suggestion
If `agoric` is not found, then you may need to add the parent directory of where your `yarn link-cli` command created a folder. For example, if you created a folder called `Agoric` in your `~/bin` directory, you may need to add `~/bin` to your PATH.
```

**Review by dckc — CHANGES_REQUESTED** [review]
I'd rather not add a whole section to the quickstart for $PATH. Please move that down into the expanded directions.

File: main/getting-started/before-using-agoric.md (hunk 3 above)
Comment (cboydstun) [reply main/getting-started/before-using-agoric.md]: sounds good! thanks!

File: main/getting-started/before-using-agoric.md (hunk 4 above)
Comment (cboydstun) [reply main/getting-started/before-using-agoric.md]: you're right one command adds it to the shell and then it'll disappear on restart... adding it permanently to your shell by modifying the ".bash_profile" file requires a restart.

File: main/getting-started/before-using-agoric.md (hunk 2 above)
Comment (dckc) [reply main/getting-started/before-using-agoric.md:19]: How was this resolved? I'm struggling to find it.

File: main/getting-started/before-using-agoric.md (hunk 4 above)
Comment (dckc) [reply main/getting-started/before-using-agoric.md]: right... discussion of adding it permanently should go in the expanded section on PATH handling... which... btw... where did it go?

Comment (cboydstun) [reply main/getting-started/before-using-agoric.md]: sorry i removed the expanded section on PATH handling based on the comment that it was [somewhat awkward](https://github.com/Agoric/documentation/pull/714/files/c2282953f685c7cba759ae42bb221d0057a16a46#r965208761) so I consolidated and married them... you'd like me to revive that section and place it near the bottom with the expanded "Install `agoric` CLI" instructions?

Comment (dckc) [reply main/getting-started/before-using-agoric.md]: That was what i was suggesting, yes; I'm not sure it's a good suggestion, though. I'm juggling too much to give this the attention it deserves just now.

Any thoughts, @kennyrowe?

Comment (cboydstun) [reply main/getting-started/before-using-agoric.md]: Any opinions? I've got a video demonstrating [install here](https://youtu.be/Yp76WxbaNhQ).  [@kennyrowe ](https://github.com/kennyrowe) @turadg @dckc

Comment (dckc) [reply main/getting-started/before-using-agoric.md]: This is to replace the existing video explainer? I don't see a change to that effect in the PR.
The existing video has voice-over; I'd rather not lose that.

File: main/getting-started/before-using-agoric.md [hunk 6]
```diff
@@ -15,42 +16,43 @@ git clone https://github.com/Agoric/agoric-sdk
...
+- To open a terminal on MacOS, see **Applications>Utilities>terminal** in the **Finder**.
+- To launch a terminal on Linux, use the **Terminal** application.
+- To access WSL from Windows, visit the [WSL documentation](https://docs.microsoft.com/en-us/windows/wsl/).
+- To use a Docker container, see the [Docker documentation](https://docs.docker.com/) and the [Agoric container images](https://hub.docker.com/u/agoric).
 
 ## Install Node.js 14.15.0 or higher
 
-Download from [nodejs.org](https://nodejs.org/) and follow the instructions for your platform.
-
+Download from [NodeJS](https://nodejs.org/) and follow the instructions for your platform.
```
Comment (dckc) [inline main/getting-started/before-using-agoric.md:55]: They call themselves "Node.js"; any reason not to use that name?

File: main/getting-started/before-using-agoric.md [hunk 7]
```diff
@@ -15,42 +16,43 @@ git clone https://github.com/Agoric/agoric-sdk
...
-   2. Click the address bar while in that folder.
-   3. Type <code>bash</code> in the address bar and press <b>Enter</b>
+<a href="https://www.apple.com/macos/">MacOS</a>,
+<a href="https://docs.microsoft.com/en-us/windows/wsl/">Windows Subsystem for Linux (WSL)</a>, and
+<a href="https://www.docker.com/">Docker</a>.
 
+- To open a terminal on MacOS, see **Applications>Utilities>terminal** in the **Finder**.
+- To launch a terminal on Linux, use the **Terminal** application.
+- To access WSL from Windows, visit the [WSL documentation](https://docs.microsoft.com/en-us/windows/wsl/).
+- To use a Docker container, see the [Docker documentation](https://docs.docker.com/) and the [Agoric container images](https://hub.docker.com/u/agoric).
```
Comment (dckc) [inline main/getting-started/before-using-agoric.md:51]: Based on discussion with @michaelfig , I'm not sure we want to support customers in using those docker images for development. We sort of supported them during our beta release for clients / wallets for folks that didn't want to install the SDK. But these docs are not for that audience.

File: main/getting-started/before-using-agoric.md (hunk 4 above)
Comment (cboydstun) [reply main/getting-started/before-using-agoric.md]: i haven't slotted in the video... it's not deployed on Agoric channels yet that sample is on my personal YouTube set to private unless you have the link.

Comment (cboydstun) [reply main/getting-started/before-using-agoric.md]: @dckc I've amended the instructions - think we could resolve this conversation, push these changes across, and then I can create an up to date video with voiceover reflecting these awesome changes we've made? =)

File: main/getting-started/before-using-agoric.md [hunk 8]
```diff
@@ -11,46 +12,48 @@ If you're familar with JavaScript development tools such as `node`, `yarn`, and
...
-agoric --version
+yarn link-cli ~/bin/agoric # creates an Agoric directory to reference
+agoric --version # should print the version number of the SDK
 ```
 
-Then proceed to [starting a project](/getting-started/start-a-project.md).
+Now you are ready proceed to [starting a project](/getting-started/start-a-project.md).
 
-A more detailed explanation follows.
+If `agoric` is not found, then you may need to add the parent directory of where your `yarn link-cli` command created a folder. For example, if you created a folder called `Agoric` in your `/bin` directory, you may need to add `/bin` to your PATH.
```
[the remark this answers is not in the document]
Comment (dckc) [reply main/getting-started/before-using-agoric.md:25]: also (less citical):


Should we tell them _how_? as in `export PATH=~/bin:$PATH`? and/or link to some widely-used docs on the topic?

 - [Linux add to $PATH: Fix "command not found" error \(Linux & Mac\) \- YouTube](https://www.youtube.com/watch?v=gkqsLRDnqlA)

File: main/getting-started/before-using-agoric.md [hunk 9]
```diff
@@ -11,46 +12,48 @@ If you're familar with JavaScript development tools such as `node`, `yarn`, and
 ```shell
 node --version # 14.15.0 or higher
 npm install --global yarn
-git clone https://github.com/Agoric/agoric-sdk
+git clone -b community-dev https://github.com/Agoric/agoric-sdk # "community-dev" branch
```
Comment (dckc) [inline main/getting-started/before-using-agoric.md:15]: The scope of this PR is drifting. Please be sure to keep the title and description up to date.

Alternatively, make a separate PR regarding `-b community-dev`.

**Review by dckc — CHANGES_REQUESTED** [review]
The `/bin` vs `~/bin` fix is critical.

File: main/getting-started/before-using-agoric.md [hunk 10]
```diff
@@ -121,3 +122,9 @@ To check that it's installed correctly:
 ```shell
 agoric --version # v0.16.0 "community-dev" branch
 ```
+
+If the install was successful, you are ready to proceed to [starting a project](/getting-started/start-a-project.md).
+
+Should `agoric` not be found, then you may need to add the parent directory of where your `yarn link-cli` command created a "agoric" directory.
+
+Please see the [Linux add to \$PATH: Fix "command not found" error (Linux & Mac) - YouTube](https://www.youtube.com/watch?v=gkqsLRDnqlA) for more information.
```
Comment (dckc) [inline main/getting-started/before-using-agoric.md:130]: It's a little odd that we discuss adding to PATH only in the quickstart section. The premise of the quickstart section is that it's for people who know the basics.

Adding a `export PATH=~/bin:$PATH` line in the code box seems more straightforward. Then we could omit these two paragraphs or move them down where we have "Note: Run echo $PATH to see your current $PATH."

not critical I suppose.

**Review by dckc — APPROVED** [review]
one more suggestion; not critical

File: main/getting-started/before-using-agoric.md (hunk 10 above)
Comment (cboydstun) [reply main/getting-started/before-using-agoric.md:130]: thanks for approving the PR! regardless of the debate I think it's in superior condition! =) 

so if look we look at the greats like installing Node.js then it automatically links for us. Also, our VS Code extension is handling it beautifully. Ideally, I'd like to see us get to a place in which the `yarn link-cli` command is optional. 

That said, the instructions with an `export PATH=~/bin:$PATH` creates a new "bin" directory and adds more complexity to the $PATH. It's messy. After doing this several times, I think running `echo $PATH`, finding your local bin, and adding `agoric` in that location is optimal imo.

Comment (dckc) [reply main/getting-started/before-using-agoric.md:130]: > After doing this several times, I think running echo $PATH, finding your local bin, and adding agoric in that location is optimal imo.

Then shouldn't the quickstart section do likewise?
